invoices/invoiceregistry_test.go

```diff
@@ -240,6 +241,69 @@ func TestSettleInvoice(t *testing.T) {
 	}
 }
 
+// Tests that invoice is rejected after expiration.
+func TestInvoiceExpiry(t *testing.T) {
```

Comment:
Can use `t.Parallel()` here, since the test doesn't interact with any global state.

Reply:
done